R/view_config_val_errors.R

```diff
@@ -80,10 +80,20 @@ clean_error_df <- function(errors_tbl) {
   if (is.null(errors_tbl)) {
     return(NULL)
   }
+
+  # Move any custom error messages to the message column
```

Comment:
I discovered I could add more relevant custom error messages to the schema. These appear in the `parentSchema$errorMessage` field so just moving it to the main message column.